core/kernel/asan.c

```diff
@@ -413,9 +413,16 @@ void __asan_register_globals(struct asan_global *globals, size_t size)
 {
 	size_t n;
```

Comment:
Since you're changing this function, please zero-initialize `n`.

Reply:
Done